document/sections/03a-motivation.adoc

```diff
@@ -0,0 +1,96 @@
+ If one desires digital, automated processing or analysis of 3D spatial questions, spatial information must be structured in an explicitly machine-readable format. When this is done, initiatives based on federated systems with heterogeneous datasets can obtain computer-generated answers to spatial 3D queries without requiring conversion. This highlights the importance of semantic alignment and standardized geometry processing in machine-driven analysis. <<Göbel2024>>
+
+ When performing federated queries on heterogeneous spatial datasets without conversion, a common vocabulary is required to semantically define the 3D object and its 3D spatial function. For both humans and machines, it must be unambiguous what is meant by a query such as:
+
+ “Does the '3D geometry' of the proposed construction '3D-intersect' with the '3D geometry'of an existing asset owned by another party?”
+
+ It should not matter whether the data is provided in a 3D-GIS, 3D-BIM or 3D-CG standard vocabulary.  
+
+=== Added value of 3D in the GeoSPARQL Vocabulary for Stakeholders
+==== We aim to support data Consumers by:
+ - access and understanding of 3D-geometry and 3D-topology from multiple sources; 
+ - the integration of 3D-data geospatial and non-geospatial;
+ - data quality evaluation of the given 3D-data;
+ - Geometric and topological description in natural language.
+
+==== We aim to support Data Providers by:
+ - Maximize the use of 3D-geometry and 3D-topology data;
+ - Defining multiple 3D topology or geometries;
+ - To quality-assure 3D geospatial data.
+
+
+=== Semantic-functional value of 3D-GeoSPARQL
+ Value for data reasoning, formal description, and semantic interoperability
+
+==== Meaningful creation and querying of 3D geometry
+ Whether manually or programmatically defining 3D geometries in BIM, GEO or CG environments, the GeoSPARQL standard could provide or connect to vocabulary to represent 3D structures. A 3D cube, for example, can be represented in multiple ways:
+
+ - As a set of bounding points, lines, and faces;
+ - As a base surface with an extrude function;
+ - As an 3D geometric primitive (e.g., the concept of a 'cube' as a spatial figure).
+
+ When GeoSPARQL offers the means to define and link 3D geometries in these different ways, it allows both humans and machines to interpret and interact with the data effectively.
```

Comment:
```suggestion
 Once GeoSPARQL offers the means to define and link 3D geometries in these different ways, it will allow both humans and machines to interpret and interact with the data effectively.
```